### PR TITLE
feat(Icon): add StarFilledCircleIcon

### PR DESCRIPTION
**Pull request by casey-speer**
### Overview

Adds the `StarFilledCircleIcon` to `gamut-icons`

### PR Checklist

- [x] Related to designs:https://www.figma.com/file/IGWuvtcKcBcrCBg100CemB/Quizzes-%26-Practice?node-id=1844%3A26258
- [x] I have run this code to verify it works

#### Testing Instructions

- Run storybook and verify `StarFilledCircleIcon` is shown in the regular icons section

### CODE REVIEW

File: packages/gamut-icons/src/svg/regular/star-filled-circle-icon.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path fill-rule="evenodd" clip-rule="evenodd" d="M12 24c6.627 0 12-5.373 12-12S18.627 0 12 0 0 5.373 0 12s5.373 12 12 12zm5.503-13.99a.752.752 0 01.297.804.75.75 0 01-.239.381l-2.597 2.153a.25.25 0 00-.078.271l1.09 3.268a.75.75 0 01-1.157.841l-2.671-1.959a.25.25 0 00-.296 0l-2.673 1.96a.75.75 0 01-1.155-.842l1.089-3.266a.25.25 0 00-.077-.271L6.444 11.2a.75.75 0 01.481-1.325H9.97a.25.25 0 00.235-.168l1.09-3.088.002-.008a.75.75 0 011.405 0l1.093 3.096a.25.25 0 00.236.167h3.043c.154 0 .303.048.429.136z" fill="#000"/>
```
Comment (dreamwasp) [inline packages/gamut-icons/src/svg/regular/star-filled-circle-icon.svg:2]: i think you're going to need to change the fill here to `currentColor` to let the icon play nicely with taking a `color` prop

Comment (casey-speer) [reply packages/gamut-icons/src/svg/regular/star-filled-circle-icon.svg]: Fixed

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@7.7.1-alpha.c173a4.0<br/>@codecademy/gamut-kit@0.6.201-alpha.c173a4.0<br/>@codecademy/gamut-labs@33.3.5-alpha.c173a4.0<br/>@codecademy/gamut@50.6.5-alpha.c173a4.0<br/>@codecademy/monaco@0.6.87-alpha.c173a4.0<br/>@codecademy/styleguide@61.5.8-alpha.c173a4.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6320f10ea8656638af070a31--gamut-preview.netlify.app>https://6320f10ea8656638af070a31--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6320f10ea8656638af070a31>Deploy Logs</a></p>

**Review by dreamwasp — APPROVED** [review]

**Review by BandanaKM — APPROVED** [review]

**Pull request merged**